sdk/programmableconnectivity/azure-programmableconnectivity/README.md

```diff
@@ -0,0 +1,172 @@
+
+
+# Azure Programmableconnectivity client library for Python
+Azure Programmable Connectivity (APC) is a cloud service that simplifies access to programmable networks across various operators and regions. With APC, developers can seamlessly integrate network API services from multiple mobile operators into their applications, ensuring a consistent Azure experience despite underlying network changes.
+
+## Getting started
+
+### Installating the package
+
+```bash
+python -m pip install azure-programmableconnectivity
+```
+
+#### Prequisites
+
+- Python 3.8 or later is required to use this package.
+- You need an [Azure subscription][azure_sub] to use this package.
+- An existing Azure Programmableconnectivity instance.
+#### Create with an Azure Active Directory Credential
+To use an [Azure Active Directory (AAD) token credential][authenticate_with_token],
+provide an instance of the desired credential type obtained from the
+[azure-identity][azure_identity_credentials] library.
+
+To authenticate with AAD, you must first [pip][pip] install [`azure-identity`][azure_identity_pip]
+
+After setup, you can choose which type of [credential][azure_identity_credentials] from azure.identity to use.
+As an example, [DefaultAzureCredential][default_azure_credential] can be used to authenticate the client:
+
+Set the values of the client ID, tenant ID, and client secret of the AAD application as environment variables:
+`AZURE_CLIENT_ID`, `AZURE_TENANT_ID`, `AZURE_CLIENT_SECRET`.
+
+Set you Apc Gateway Id as `APC_GATEWAY_ID`.
+
+Use the returned token credential to authenticate the client:
+
+```python
+>>> from azure.programmableconnectivity import ProgrammableConnectivityClient
+>>> from azure.identity import DefaultAzureCredential
+>>> client = ProgrammableConnectivityClient(endpoint='<endpoint>', credential=DefaultAzureCredential())
+```
+
+or you can explicitly give the variables
+
+```python
+from azure.identity import ClientSecretCredential
+credential = ClientSecretCredential(
+    tenant_id=<AZURE_TENANT_ID>,
+    client_id=<AZURE_CLIENT_ID>,
+    client_secret=<AZURE_CLIENT_SECRET>,
+)
+client = ProgrammableConnectivityClient(endpoint='<endpoint>', credential=credential)
+```
+
+## Examples
+
+### Pre-requisites
```

Comment:
This header seems misplaced

Reply:
I'll rename it. It is meant to mean that all of the examples use that bit of code.